cmake/emu/renode.cmake

```diff
@@ -11,11 +11,18 @@ set(RENODE_FLAGS
   --pid-file renode.pid
   )
 
+# Check if there are any Renode script overlays defined for the target board
+set(resc_overlay_file ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.resc)
+if(EXISTS ${resc_overlay_file})
```

Comment:
This mixes "overlays" and "overlay" - overlays implies multiple but this code does not. I'm not familiar with renode, so should this just support a singular overlay or would it want/need to support multiple?

Reply:
This should support a singular overlay, so the comment needs to be reworded. As for supporting multiple overlays this would not be very useful as, by design, the overlay is for a specific combination of a sample/test and a board to add additional Renode options so that everything is simulated correctly.

Reply:
Changed the comment to specify a single overlay.